submissions/krylenger/dom-api-task/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Motorcycles</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <input id='nav__toggle' type="checkbox" class="nav__toggle">
+    <label for="nav__toggle" class="nav__toggle-label"></label>
+    <nav class="nav">
+        <ul class="nav__ul">
```

Comment:
I'd recommend to generate this navigation from your data as well in order to make your app more dynamic

Reply:
✅ done
is it the way we should do it in real work as well? or this is just to practice more in adding content dynamically?
my concern that it can affect the perfomance of website during first load. 
or the difference is minor and doesn't affect in general?

Reply:
and is it better to load content like this: 
`document.addEventListener('DOMContentLoaded', loadNavLi)`
or like this: 
`loadNavLi()`
? 
or this is doesn't matter in term of perfomance?

Reply:
I'd recommend using some `initApp` function and place there all event listeners and different 'load' functions. Then you can pass this function to `document.addEventListener('DOMContentLoaded', initApp)`

Reply:
thanks,
✅ Done